skafka/src/main/scala/com/evolutiongaming/skafka/ConfigHelpers.scala

```diff
@@ -0,0 +1,60 @@
+package com.evolutiongaming.skafka
+
+import com.evolutiongaming.config.ConfigHelper.{ConfigOps, FromConf}
+import com.typesafe.config.{Config, ConfigException, ConfigRenderOptions, ConfigValue}
+
+import java.nio.file.Path
+import scala.concurrent.duration.{Duration, FiniteDuration, MILLISECONDS, SECONDS, TimeUnit}
+import scala.jdk.CollectionConverters.SetHasAsScala
+import scala.util.{Failure, Success, Try}
+
+object ConfigHelpers {
+
+  type Pair = (String, String)
+
+  implicit val ClassFromConf: FromConf[Class[_]] = FromConf[Class[_]] { (conf, path) =>
+    val className = conf.getString(path)
+    Try(Class.forName(className)) match {
+      case Failure(_)     => throw new ConfigException.BadValue(conf.origin(), path, s"Class '$className' doesn't exist")
```

Comment:
Shouldn't we use pure functions only?

Reply:
Maybe, but it's the way, how the other conversion implicits in skafka were written, so I choose to maintain this style.